lib/utils.js

```diff
@@ -776,3 +815,22 @@ exports.stackTraceFilter = function() {
 exports.isPromise = function isPromise(value) {
   return typeof value === 'object' && typeof value.then === 'function';
 };
+
+/**
+ * Given integer `int`, try to convert it to a hex string.
+ * @param {number} int Integer
+ * @returns {string} Hexadecimal representation thereof
+ */
+exports.toHex = function toHex(int) {
+  return '0x' + int.toString(16).toUpperCase();
```

Comment:
This doesn't have any validation compared to other functions.`toHex('foo')` will happily return `0xFOO`. Is that expected behavior?